src/libraries/System.Security.Cryptography.Pkcs/src/System.Security.Cryptography.Pkcs.csproj

```diff
@@ -161,152 +160,80 @@ System.Security.Cryptography.Pkcs.EnvelopedCms</PackageDescription>
-    <Compile Include="$(CommonPath)Interop\Windows\Crypt32\Interop.CryptMsgParamType.cs"
-             Link="Common\Interop\Windows\Crypt32\Interop.CryptMsgParamType.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\Crypt32\Interop.CryptMsgType.cs"
-             Link="Common\Interop\Windows\Crypt32\Interop.CryptMsgType.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\Crypt32\Interop.CryptMsgUpdate.cs"
-             Link="Common\Interop\Windows\Crypt32\Interop.CryptMsgUpdate.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\Crypt32\Interop.CryptRc2Version.cs"
-             Link="Common\Interop\Windows\Crypt32\Interop.CryptRc2Version.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\Crypt32\Interop.DATA_BLOB.cs"
-             Link="Common\Interop\Windows\Crypt32\Interop.DATA_BLOB.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\Crypt32\Interop.ErrorCode.cs"
-             Link="Common\Interop\Windows\Crypt32\Interop.ErrorCode.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\Crypt32\Interop.FindOidInfo.cs"
-             Link="Common\Interop\Windows\Crypt32\Interop.FindOidInfo.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\Crypt32\Interop.MsgControlType.cs"
-             Link="Common\Interop\Windows\Crypt32\Interop.MsgControlType.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\Crypt32\Interop.MsgEncodingType.cs"
-             Link="Common\Interop\Windows\Crypt32\Interop.MsgEncodingType.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.FormatMessage.cs"
-             Link="Common\Interop\Windows\kernel32\Interop.FormatMessage.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.Heap.cs"
-             Link="Common\Interop\Windows\kernel32\Interop.Heap.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\NCrypt\Interop.ErrorCode.cs"
-             Link="Common\Interop\Windows\NCrypt\Interop.ErrorCode.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\NCrypt\Interop.Properties.cs"
-             Link="Common\Interop\Windows\NCrypt\Interop.Properties.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\NCrypt\Interop.NCryptFreeObject.cs"
-             Link="Common\Interop\Windows\NCrypt\Interop.NCryptFreeObject.cs" />
-    <Compile Include="$(CommonPath)Interop\Windows\Interop.Libraries.cs"
-             Link="Common\Interop\Windows\Interop.Libraries.cs" />
+    <Compile Include="$(CommonPath)Interop\Windows\Advapi32\Interop.CryptAcquireContext.cs" Link="Common\Interop\Windows\Advapi32\Interop.CryptAcquireContext.cs" />
+    <Compile Include="$(CommonPath)Interop\Windows\Advapi32\Interop.CryptDestroyHash.cs" Link="Common\Interop\Windows\Advapi32\Interop.CryptDestroyHash.cs" />
+    <Compile Include="$(CommonPath)Interop\Windows\Advapi32\Interop.CryptDestroyKey.cs" Link="Common\Interop\Windows\Advapi32\Interop.CryptDestroyKey.cs" />
```

Comment:
Is it really necessary to restyle the csproj?  And, if so, is it necessary/appropriate to do it as part of this change?

Reply:
This keeps being made by VS automatically at some point in the workflow when having the projects open. Will undo.